test/pylib/scylla_cluster.py

```diff
@@ -838,6 +852,24 @@ def setLogger(self, logger: logging.LoggerAdapter):
         for srv in self.servers.values():
             srv.setLogger(self.logger)
 
+    async def change_ip(self, server_id: ServerNum) -> ActionReturn:
+        """Lease a new IP address and update conf/scylla.yaml with it. The
+        original IP is released at the end of the test to avoid an
+        immediate recycle within the same cluster."""
```

Comment:
We should probably test immediate recycle at some point too.
But we shouldn't hope for this to work until we start identifying nodes by UUID everywhere and rejecting any messages we receive whose destination UUID doesn't match ours.
Just noting for the future.